prime-router/src/main/kotlin/TranslatorConfiguration.kt

```diff
@@ -57,7 +57,7 @@ abstract class TranslatorConfiguration(val type: String) : TranslatorProperties
  */
 data class Hl7Configuration
 @JsonCreator constructor(
-    // deprecated, please don't use
+    override val  schemaName: String = HL7_SCHEMA,
```

Comment:
Looks like there is extra spacing between `val` and `schemaName`. Make sure to install ktlint and use that to enforce code standards.